src/catchup/IndexBucketsWork.h

```diff
@@ -20,7 +20,7 @@ template <class BucketT> class IndexBucketsWork : public Work
     class IndexWork : public BasicWork
     {
         std::shared_ptr<BucketT> mBucket;
-        std::unique_ptr<typename BucketT::IndexT const> mIndex;
+        std::unique_ptr<typename BucketT::IndexT const> mIndex{nullptr};
```

Comment:
I don't understand the motivation behind this change; it's a no-op and IMHO doesn't add any clarity to the code (it's also weird that we don't explicitly initialize shared_ptr above). I think it's ok to omit initializers for the std containers as it's obvious that they have a sane default constructor (unlike for the custom or primitive types)